cmake/configs/aarch64/linux/config.cmake

```diff
@@ -13,7 +13,11 @@ set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -D_GNU_SOURCE" CACHE STRING "" FORCE)
 set(WT_BUFFER_ALIGNMENT_DEFAULT "4096" CACHE STRING "")
 
 # ARMv8-A is the 64-bit ARM architecture, turn on the optional CRC instructions.
```

Comment:
Recommend this comment is updated to reflect the differing options.

Reply:
3d2d06ba6